scripts/dashboard/validate_dashboards_format.py

```diff
@@ -0,0 +1,84 @@
+import json
```

Comment:
Add a file docstring that explains how this script is to be used, including the command line arguments.

Reply:
done.